griptape/tools/google_docs/tool.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import annotations
+import logging
+from attr import field, define
+from schema import Schema, Literal, Or
+from attr import define
+from griptape.artifacts import ErrorArtifact, InfoArtifact, BlobArtifact, ListArtifact
+from griptape.utils.decorators import activity
+from griptape.tools import BaseGoogleClient
+from googleapiclient.discovery import Resource
+
+
+@define
+class GoogleDocsClient(BaseGoogleClient):
+    owner_email: str = field(default=None)
+    DOCS_SCOPES = ['https://www.googleapis.com/auth/documents']
+    DRIVE_SCOPES = ['https://www.googleapis.com/auth/drive.file']
+
+    def _build_client(self, scopes: list[str], version: str, tool: str) -> Resource:
```

Comment:
Let's move protected method to the end of the class.

Reply:
Done!